markdown/en/documentation/configuration.md

```diff
@@ -1,8 +1,28 @@
 ## Configuration
 
 ---
-This chapter describes the minimum configuration necessary to search and retrieve results from FACT-Finder.
+This chapter describes the minimal configuration necessary to search and retrieve results from FACT-Finder. You can configure the communication via the custom element `ff-communication` or directly via JavaScript.
 
+### A minimal configuration consists of the following settings:
+
+#### **url**
+Specify the FACT-Finder endpoint. Please note the context name `/FACT-Finder-7.2`. It is not sufficient to provide the top-level domain like `http://web-components.fact-finder.de/` which will result in an error.
+
+**Note:** Make sure you are using the same protocol (http://, https://) for the `url` setting and webapp. Using `url="http://web-components.fact-finder.de/FACT-Finder-7.2"` on a page hosted via **HTTPS** will result in an error.
+
+---
+
+#### **version**
+Specify the FACT-Finder version. E.g. `7.2` or  `7.3`.
+
+---
+
+#### **channel** 
+Specify the channel name which is used for the search. You can check which channels are available at the FACT-Finder UI.
```

Comment:
`Specify the name of the channel you want to use for the search. You can find the available channels in the FACT-Finder UI.`